docs/faq.md

```diff
@@ -0,0 +1,157 @@
+# Feldera Frequently Asked Questions
+
+## What is Feldera
+
+Feldera is an early-stage start-up building a new data processing
+engine for *streaming data*.  In the rest of this document we will
+write "F" for Feldera.
+
+Using indistry jargon, the F engine can be described in several ways:
+
+- a continuous query engine
+- a standing query engine
+- performing incremental view maintenance
+- implementing materialized views
+
+## What is streaming data?
```

Comment:
This one is interesting.  There's a lot of confusion around this and maybe we should use the opportunity to clarify things a bit. I believe that "streaming" simply means that data arrives continuously. So if we limit ourselves to streaming data, we exclude other data transfer modes, like events and data that arrives in periodic batches, both of which we support.

In addition, the term "streaming" focuses on how data moves as opposed to what it means. And what it means is changes (insertions/deletions/update) to a data set.

So I would define Feldera as a query engine for changing data.  Changes can arrive as events, batches, or continuous streams. People often focus on the latter, and use the term "streaming analytics". While Feldera is indeed a best-in-class streaming analytics engine, it supports the other data transfer modes as well, and can even work with a mix of event, batch, and stream data.